Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/RootSearchSection.java

```diff
@@ -70,6 +70,11 @@ protected String getSessionKey() {
     return SEARCH_SESSIONKEY;
   }
 
+  // For child sections that need to skip new Search UI, override this function and return false.
+  public boolean useNewSearch() {
```

Comment:
Should this perhaps just be `protected`?

```suggestion
  protected boolean useNewSearch() {
```

It's not intended to be used a public interface, just for internal operations.